src/EA.Weee.Web/Areas/AatfReturn/Views/Returns/Index.cshtml

```diff
@@ -57,84 +61,91 @@
+                                    if (item.ReturnsListRedirectOptions.RedirectTaskList)
+                                    {
+                                        @Html.NavigationRouteLink("Continue", "Continue return", AatfRedirect.Default, new { action = "Index", controller = "AatfTaskList", returnId = @item.ReturnViewModel.ReturnId }, null)
+                                    }
+                                    else if (item.ReturnsListRedirectOptions.RedirectSelectYourPcs)
+                                    {
+                                        @Html.NavigationRouteLink("Continue", "Continue return", AatfRedirect.SelectPcsRouteName, new { action = "Index", returnId = @item.ReturnViewModel.ReturnId, organisationId = Model.OrganisationId }, null)
+                                    }
+                                    else if (item.ReturnsListRedirectOptions.RedirectReportingOptions)
+                                    {
+                                        @Html.NavigationRouteLink("Continue", "Continue return", AatfRedirect.SelectReportOptionsRouteName, new { action = "Index", returnId = @item.ReturnViewModel.ReturnId, organisationId = Model.OrganisationId }, null)
+                                    }
                                 }
                             }
-                        }
-                    </td>
-                    <td class="govuk-table__cell prsd-table-collapse prsd-table-collapse-cell">
-                        @Html.NavigationRouteLink("Summary", "Return summary", AatfRedirect.Default, new { action = "Index", controller = "ReturnsSummary",  returnId = @item.ReturnViewModel.ReturnId }, null)
-                    </td>
-                </tr>
-            }
+                        </td>
+                        <td class="govuk-table__cell prsd-table-collapse prsd-table-collapse-cell">
+                            @Html.NavigationRouteLink("Summary", "Return summary", AatfRedirect.Default, new { action = "Index", controller = "ReturnsSummary", returnId = @item.ReturnViewModel.ReturnId }, null)
+                        </td>
+                    </tr>
+                }
 
             </tbody>
         </table>
+    }
+    else
+    {
         <p>
-            @Html.ActionLink("Perform another activity", "ChooseActivity", "Home", new { area = "Scheme", pcsId = @Model.OrganisationId }, null)
+            No submissions to report
         </p>
-        <style type="text/css">
-
-            @@media only screen and (max-width: 760px), (min-device-width: 768px) and (max-device-width: 1024px) {
-                td:nth-of-type(1):before {
-                    content: "Year";
-                }
+    }
+    <p>
+        @Html.ActionLink("Perform another activity", "ChooseActivity", "Home", new { area = "Scheme", pcsId = @Model.OrganisationId }, null)
+    </p>
+    <style type="text/css">
```

Comment:
Can this css be moved to the SASS files?

Reply:
@JohnPadgettCivica  How do we go about moving this to the SASS file? Not sure what this css is doing.....

Reply:
We dont need to move this to SASS as its specific to this page and table

Reply:
Awesome!